templates/web/sutton/header_logo.html

```diff
@@ -1,2 +1,3 @@
-<a href="https://www.kingston.gov.uk/" id="site-logo">[% site_name %]</a>
+<a href="https://www.sutton.gov.uk/" id="site-logo">[% site_name %]</a>
 <a href="/waste" id="report-cta" title="[%- loc('Report a problem') -%]">[%- loc('Report') -%]</a>
+<label id="nav-link" for="main-nav-btn" aria-expanded="false">[% loc('Main Navigation') %]</label>
```

Comment:
Is there a reason this line isn't in header_site.html to match base? just so it's easier to see what has changed from base to the cobrand.

Reply:
Fair point. I just amend this and moved it back to header_site.html.
Thanks Matthew